packages/viewer/src/components.d.ts

```diff
@@ -4,424 +4,345 @@
  * This is an autogenerated file created by the Stencil compiler.
```

Comment:
This file appears to update for me running the dev server. Unsure if we've just made changes + not run the dev server prior to this change, so I've left it in for now

Reply:
Must be a change with the Stencil 2 update.